jre.go

```diff
@@ -149,3 +162,34 @@ func (j JRE) Contribute(layer libcnb.Layer) (libcnb.Layer, error) {
 func (j JRE) Name() string {
 	return j.LayerContributor.LayerName()
 }
+
+func (j JRE) buildCustomJRE(layerPath string) error {
+	buildpackPath := filepath.Dir(layerPath)
+	customDir := filepath.Join(buildpackPath, "custom-jre")
+
+	def := fmt.Sprintf("--add-modules ALL-MODULE-PATH --no-man-pages --no-header-files --strip-debug --output %s", customDir)
```

Comment:
We might want to add `--compress=1` by default. It'll shave off another 20-30M. From what I have read, it does come with a slight cost of increased startup time (<10ms) but given the context of buildpacks and typically long running apps, that seems like a reasonable trade-off.